src/pages/workspace/components/ActivityDetails/ActivityLinks.tsx

```
@@ -60,40 +157,67 @@ const ActivityLinks = ({ viewMode }: ActivityLinksProps) => {
-            ellipsis={{ rows: 2 }}
-            style={{ marginBottom: 5, fontSize: 13, lineHeight: 1.2 }}
-          >
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod cupiditate dolores...
-          </Paragraph>
-          <Paragraph
-            ellipsis={{ rows: 2, expandable: false }}
-            style={{ fontSize: 13, marginBottom: 0 }}
+      {(listLink || []).map((item, index) => (
+        <div style={{ position: 'relative' }}>
+          <Card
+            key={index}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)